README.md

```diff
@@ -27,7 +27,13 @@ buffer names.
 
 ### Installation
 
-Install the `ivy` package from MELPA / GNU ELPA.
+Install the `counsel` package preferably from MELPA. If you prefer to
```

Comment:
Why suggest `counsel` rather than `ivy`? `ivy` is the core completion package; `counsel` is just a useful library built on top of it.

Also, as a FSF-copyrighted project distributed via GNU ELPA, MELPA should always come second to GNU ELPA, if we even mention it at all.

Nit: Please also add two spaces after a full stop, as per the `sentence-end-double-space` setting in `.dir-locals.el`.

Reply:
I was trying to get `counsel-find-file`, I quickly read the first paragraph [here](https://oremacs.com/swiper/#installing-from-emacs-package-manager) which seemed to suggest that the best thing to do was to install counsel. However, I did not read further than that first paragraph because a quick look at that subsection made me assume that these were only more detailed instructions for how to install counsel. But now I see it actually also mentions that ivy can be installed separately.